controlpanel/frontend/views/reset.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+from django.urls import reverse_lazy
+from django.contrib import messages
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.views.generic.edit import FormView
+from controlpanel.frontend.forms import ResetHomeDirectoryForm
+from controlpanel.frontend.consumers import start_background_task
+
+
+class ResetHome(LoginRequiredMixin, FormView):
+    """
+    A view to help automate the resetting of the user's home directory. To be
+    used when, for example, their conda or r-studio deployment gets into a bad
+    state. See the related helm chart (TBD) for the details of what actually
+    happens in the user's home directory.
+    """
+
+    template_name = "reset.html"
+    form_class = ResetHomeDirectoryForm
+
+    def get_success_url(self):
+        """
+        The form is valid, so kick off the background job to run the helm
+        chart and signal to the user the reset is underway.
+        """
+        start_background_task(
+            "home.reset",
+            {
+                "user_id": self.request.user.id,
```

Comment:
Random info follows:

FYI: In this case the background job doesn't seem to do anything with the k8s API so we technically don't need to pass the ID Token to the worker.

But so that you know, in general if the background consumer needs to interact with the k8s API using the user credentials it would need the ID Token and it would not be able to retrieve it as it's in a separate process from the frontend.

e.g. they can't call something like `self.request.user` as this is in a different context.

Random info but I thought it may be useful for you to know for the future 😬